src/tools/compiletest/src/runtest/run_make.rs

```diff
@@ -517,14 +517,13 @@ impl TestCx<'_> {
 
         let proc = disable_error_reporting(|| cmd.spawn().expect("failed to spawn `rmake`"));
         let (Output { stdout, stderr, status }, truncated) = self.read2_abbreviated(proc);
+        let stdout = String::from_utf8_lossy(&stdout).into_owned();
+        let stderr = String::from_utf8_lossy(&stderr).into_owned();
+        // This conditions on `status.success()` so we don't print output twice on error.
+        // NOTE: this code is called from a libtest thread, so it's hidden by default unless --nocapture is passed.
+        self.dump_output(status.success(), &cmd.get_program().to_string_lossy(), &stdout, &stderr);
```

Comment:
Nit: note that this has one issue: this will not distinguish between which `rmake.exe` for which test. We may want to either print the path to test beforehand, or we modify `rmake.exe` to the test path maybe

```
running 1 tests
------rmake.exe stdout------------------------------
meow

------rmake.exe stderr------------------------------
bark

------------------------------------------
.

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 380 filtered out; finished in 257.36ms

Build completed successfully in 0:00:10
```

Reply:
yeah, i'll fix this. good call.